internal/storage/db/internal/db.go

```diff
@@ -66,13 +68,15 @@ func NewDBStorage(ctx context.Context, db *goqu.Database, dbOpts ...DBOpt) (DBSt
 	return &dbStorage{
 		opts:                opts,
 		db:                  db,
+		logger:              zap.L().Named("db"),
 		SubscriptionManager: storage.NewSubscriptionManager(ctx),
 	}, nil
 }
 
 type dbStorage struct {
-	opts *dbOpt
-	db   *goqu.Database
+	opts   *dbOpt
+	db     *goqu.Database
+	logger *zap.Logger
```

Comment:
This doesn't need to be a struct member because it's only used in the `CheckSchema` method. Just create the logger in the method itself.